internal/runners/config/config.go

```diff
@@ -18,13 +23,103 @@ type primeable interface {
 	primer.Analyticer
 }
 
-func NewConfig(prime primeable) (*Config, error) {
-	return &Config{
+func NewList(prime primeable) (*List, error) {
+	return &List{
 		out: prime.Output(),
 		cfg: prime.Config(),
 	}, nil
 }
 
-func (c *Config) Run(usageFunc func() error) error {
-	return usageFunc()
+type configData struct {
+	Key     string `locale:"key,Key"`
+	Value   string `locale:"value,Value"`
+	Default string `locale:"default,Default"`
+}
+
+type configOutput struct {
+	out     output.Outputer
+	cfg     *config.Instance
+	options []mediator.Option
+	data    []configData
+}
+
+func (c *List) Run(usageFunc func() error) error {
+	registered := mediator.AllRegistered()
+	sort.SliceStable(registered, func(i, j int) bool {
+		return registered[i].Name < registered[j].Name
+	})
+
+	var data []configData
+	for _, opt := range registered {
+		configuredValue := c.cfg.Get(opt.Name)
+		data = append(data, configData{
+			Key:     formatKey(opt.Name),
+			Value:   formatValue(opt, configuredValue),
+			Default: formatDefault(mediator.GetDefault(opt)),
```

Comment:
For JSON output the formatting here isn't desirable. eg. string values now have double quotes.